src/api/operator/utils.ts

```diff
@@ -0,0 +1,51 @@
+// Util functions that only pertain to/deal with operator API related stuff
+
+import BigNumber from 'bignumber.js'
+
+import { FILLED_ORDER_EPSILON, ONE_BIG_NUMBER } from 'const'
+
+import { RawOrder } from './types'
+
+export type OrderStatus = 'open' | 'filled' | 'expired' | 'partially filled'
+
+function isOrderFilled(order: RawOrder): boolean {
+  let amount, executedAmount
+
+  if (order.kind === 'buy') {
+    amount = new BigNumber(order.buyAmount)
+    executedAmount = new BigNumber(order.executedBuyAmount)
+  } else {
+    amount = new BigNumber(order.sellAmount)
+    executedAmount = new BigNumber(order.executedSellAmount)
```

Comment:
For sell orders, this should be:
```suggestion
    executedAmount = new BigNumber(order.executedSellAmount).minus(order.executedFeeAmount)
```

This is because in the SC we wanted the trade event to emit the exact amount that was being transferred from the user, and I suspect the back end just returns this value without modification. @vkgnosis brought this up in an issue to change the event (which would also change the value returned by the back end) to have this changed to be the filled sell amount: https://github.com/gnosis/gp-v2-contracts/issues/392.

Reply:
I wrote in the issue, but in short I think the current approach is ok, although I have no super strong opinion about that. Is just good to know and be explicit about it. Thanks for pointing this out

Reply:
Updated and added unit tests accounting for the fee